src/components/renderer/LookupNotification.tsx

```diff
@@ -86,12 +84,19 @@ function LookupNotificationComponent({
   ])
 
   const mergeLookupData = React.useCallback(
-    (
-      newValue: unknown,
-      dataLookupResult: SubmissionTypes.S3SubmissionData['submission'],
-      elementLookupResult: FormTypes.FormElement[],
-    ) => {
-      if (elementLookupResult) {
+    ({
+      newValue,
+      dataLookupResult,
+      elementLookupResult,
+      executedLookup,
+    }: {
+      newValue: unknown
+      dataLookupResult: SubmissionTypes.S3SubmissionData['submission']
+      elementLookupResult: FormTypes.FormElement[]
+      executedLookup: ExecutedLookups
+    }) => {
+      const executedLookupResult = executedLookup?.[element.name]
+      if (elementLookupResult && executedLookupResult !== false) {
```

Comment:
Whats the intention here? We are only injecting pages if the lookup ran successfully in the past?

Reply:
That is the arg it receives, so it's the current value.  So only injecting if the lookup was successful